compute_tools/src/http/openapi_spec.yaml

```diff
@@ -591,18 +584,15 @@ components:
           type: string
           example: "1.0.0"
 
-    SafekeepersLsn:
+    ComputeSchema:
       type: object
       required:
-        - safekeepers
-        - wal_flush_lsn
+        - spec
       properties:
-        safekeepers:
-          description: Primary replica safekeepers
-          type: string
-        wal_flush_lsn:
```

Comment:
Does it not require wal_flush_lsn now? Or is that going to be in spec?

Reply:
Forgot that, it's still required, I'll fix the spec